### PR TITLE
Translate expect vs should syntax to zh-tw

### PR DESCRIPTION
**Pull request by gaga5lala**
Thanks for this awesome guide :)

In this PR, fix a typo and translate "expect vs should syntax" to zh-tw

### CODE REVIEW

File: content/zh_tw.html [hunk 1]
```diff
@@ -258,7 +259,7 @@
 </p> -->
 
 <p>
-「單一條件」意指一個測項應該只帶有一個檢查 (expection, assertion)。這樣做能幫助你直接前往失敗的測項尋找可能的問題，也讓程式碼比較好看。
+「單一條件」意指一個測項應該只帶有一個檢查 (expectation, assertion)。這樣做能幫助你直接前往失敗的測項尋找可能的問題，也讓程式碼比較好看。
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: why this change?

> The 'one expectation' tip is more broadly expressed as 'each test should make only one assertion'. This helps you on finding possible errors, going directly to the failing test, and to make your code readable.
> http://betterspecs.org/#single

File: content/zh_tw.html [hunk 2]
```diff
@@ -386,6 +387,98 @@
 
 <article>
 
+<h1><a name="expect">語法 expect 與 should</a></h1>
+
+<!-- <p>
+On new projects always use the <code>expect</code> syntax.
+</p> -->
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: Why comment out instead of replacing by translated text?

File: content/zh_tw.html [hunk 3]
```diff
@@ -386,6 +387,98 @@
...
+
+<p>
+在 RSpec3 總是使用 <code>expect</code> 語法。
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: Could you check this translation again?

> On new projects always use the expect syntax.

File: content/zh_tw.html [hunk 4]
```diff
@@ -386,6 +387,98 @@
...
+
+<div>
+<pre><code class="ruby">it 'creates a resource' do
+  expect(response).to respond_with_content_type(:json)
+end</code></pre>
+</div>
+
+<!-- <p>Configure the Rspec to only accept the new syntax on new projects, to avoid having the 2 syntax all over the place.</p> -->
+
+<p>調整 Rspec 設定, 只接受新版的語法, 避免新舊語法同時出現。</p>
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: Should use Chinese comma.

File: content/zh_tw.html [hunk 5]
```diff
@@ -386,6 +387,98 @@
...
+<div>
+<pre><code class="ruby">context 'when not valid' do
+  it { is_expected.to respond_with 422  }
+end
+</code></pre>
+</div>
+
+<!-- <p>On old projects you can use the <a href="https://github.com/yujinakayama/transpec">transpec</a> to convert them to the new syntax.</p> -->
+
+<p>可以使用 <a href="https://github.com/yujinakayama/transpec">transpec</a> 將 RSpec2 語法轉換成 RSpec3 的語法。</p>
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: Original text did not specify RSpec versions.

> On old projects you can use the transpec to convert them to the new syntax.

File: content/zh_tw.html [hunk 6]
```diff
@@ -386,6 +387,98 @@
...
+
+<!--
+<p>More about the new Rspec expectation syntax can be found <a href="http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax">here</a> and <a href="http://myronmars.to/n/dev-blog/2013/07/the-plan-for-rspec-3#what_about_the_old_expectationmock_syntax">here</a>.</p>
+-->
+
+<p>更多 RSpec3 expectation 語法請看 <a href="http://myronmars.to/n/dev-blog/2012/06/rspecs-new-expectation-syntax">這裡</a> 和 <a href="http://myronmars.to/n/dev-blog/2013/07/the-plan-for-rspec-3#what_about_the_old_expectationmock_syntax">這裡</a>。</p>
```
Comment (JuanitoFatas) [inline content/zh_tw.html]: Original text did not specify RSpec versions.

> More about the new Rspec expectation syntax can be found here and here.

Comment (JuanitoFatas) [inline content/zh_tw.html]: 這裡 和 這裡
^^^^^^^^^^^^

should without spaces in between.

File: content/zh_tw.html (hunk 1 above)
Comment (gaga5lala) [inline content/zh_tw.html]: Thanks, for your review :)

I can't find "expection" in oxford dictionary, so I think it's should be noun for expect, so I change this to "expectation"
http://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=expection

Maybe I misunderstood it's means?

File: content/zh_tw.html (hunk 3 above)
Comment (gaga5lala) [inline content/zh_tw.html]: I use this translation because in gem transpec mention that
> With Transpec you can upgrade your RSpec 2 specs to RSpec 3 in no time.

so I use the word RSpec3 instead of new projects.

File: content/zh_tw.html (hunk 2 above)
Comment (gaga5lala) [inline content/zh_tw.html]: I just do translate for a fews times, I guess temporality comment out original text can help someone reviews translate more easily.

I would remove those comment later.

Thanks for your suggestion.

File: content/zh_tw.html (hunk 5 above)
Comment (gaga5lala) [inline content/zh_tw.html]: In gem transpec mention this, so I use specify RSpec versions.
> With Transpec you can upgrade your RSpec 2 specs to RSpec 3 in no time.

File: content/zh_tw.html (hunk 6 above)
Comment (gaga5lala) [inline content/zh_tw.html]: Thanks for those suggestios i will fix it :)

File: content/zh_tw.html (hunk 3 above)
Comment (JuanitoFatas) [inline content/zh_tw.html]: I agree it is clearer. But you should change English first, then make your improvement in zh_tw version. Otherwise, this is not a correct translation. Also, how about when RSpec 4 came out? Will you update this text? English description is agnostic to RSpec version changes.

File: content/zh_tw.html (hunk 5 above)
Comment (JuanitoFatas) [inline content/zh_tw.html]: Same as https://github.com/andreareginato/betterspecs/pull/160#discussion_r49273020

File: content/zh_tw.html (hunk 1 above)
Comment (JuanitoFatas) [inline content/zh_tw.html]: I think this could be translated into:

「單一條件（one expectation）」泛指一個測項應該只有一個檢查（assertion）。

File: content/zh_tw.html (hunk 2 above)
Comment (JuanitoFatas) [inline content/zh_tw.html]: I'll leave this for you to decide : )

**Comment by gaga5lala** [comment]
Hi @JuanitoFatas , thanks for all your suggestions :)

Follow your suggestions, I push 2 new commit to fix them, one only thing I haven't change is keeping comment out original text.

**Comment by JuanitoFatas** [comment]
LGTM.

**Comment by onebree** [comment]
@JuanitoFatas thank you for reviewing this translation PR. And @gaga5lala thank you for your fixes!
